src/e2e/java/uk/gov/hmcts/reform/sscs/functional/mya/SubscriptionTest.java

```diff
@@ -14,7 +14,7 @@ public class SubscriptionTest extends BaseFunctionTest {
     @Test
     public void shouldUpdateSubscription() throws IOException, InterruptedException {
         String newUserEmail = createRandomEmail();
-        CreatedCcdCase ccdCase = createCcdCase(createRandomEmail());
+        CreatedCcdCase ccdCase = createCcdCase(newUserEmail);
```

Comment:
Why this change?

Reply:
Because it creates two random emails(two times) and then compares and functional test fails

Reply:
Why is that needed for this ticket?

Reply:
Rollbacked this failing test will see what is happen